.github/workflows/checks.yml

```diff
@@ -18,6 +18,9 @@ jobs:
         - python-version: 3.8
           env:
             TOXENV: typing
+        - python-version: 3.8
```

Comment:
I wonder if we should run it in the latest Python instead.

Reply:
Should we also run the `typing` check on the latest Python then, as I think they check the same thing? In that case we will stop checking (and so supporting) that the type hints work as expected on older Python versions and I thought it's a nice thing to support.

Reply:
(I mean, for users that use Scrapy on older Python versions and run mypy on their code)